lib/ui/LandingPage.dart

```diff
@@ -0,0 +1,240 @@
+      left: 34, right: 34);
+  final EdgeInsets footerTextEdgePadding = const EdgeInsets.only(
+      left: 30, right: 30, bottom: 16);
+
+  final double subtitleLineSpace = 13;
+  final double detailTextLineSpace = 40;
+  final double actionLineSpace = 15;
+  final int detailTextMaxLines = 3;
+  final int footerTextMaxLines = 1;
+
+  const _DefaultStyle({
+    TextStyle detailTextStyle,
+    TextStyle footerTextStyle,
+    EdgeInsets headerEdgePadding,
+    EdgeInsets detailTextEdgePadding,
+    EdgeInsets actionEdgePadding,
+    EdgeInsets footerTextEdgePadding,
+    double subtitleLineSpace,
+    double detailTextLineSpace,
+    double actionLineSpace,
+    int detailTextMaxLines,
+    int footerTextMaxLines,
+  });
+}
+
+const LandingPageStyle _defaultStyle = const _DefaultStyle();
+
+class LandingPage extends StatelessWidget {
+  final LandingPageViewModel _viewModel;
+  final LandingPageStyle _style;
+
+  const LandingPage({Key key,
+    LandingPageViewModel viewModel,
+    LandingPageStyle style = _defaultStyle})
+      : this._viewModel = viewModel,
+        this._style = style,
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SingleChildScrollView(
+        child: Column(
+          children: <Widget>[
+            _buildHeader(),
+            _buildSubtitle(),
+            SizedBox(
+              height: _style.subtitleLineSpace
+            ),
+            _buildDetailText(),
+            SizedBox(
+              height: _style.detailTextLineSpace
+            ),
+            _buildAction(context),
+            SizedBox(
+              height: _style.actionLineSpace
+            ),
+            _buildFooter(context),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Row _buildFooter(BuildContext context) {
+    return Row(
+            children: <Widget>[
+              Expanded(
+                child: Padding(
+                  padding: _style.footerTextEdgePadding,
+                  child: FlatButton(
+                    onPressed: () => _onMenuPressed(context),
+                    child: Text(
+                      _viewModel.footerText,
+                      textAlign: TextAlign.center,
+                      style: _style.footerTextStyle,
+                      maxLines: _style.footerTextMaxLines,
+                      overflow: TextOverflow.ellipsis,
+                    ),
+                  ),
+                ),
+              ),
+            ],
+          );
+  }
+
+  Padding _buildAction(BuildContext context) {
+    return Padding(
+            padding: _style.actionEdgePadding,
+            child: RoundedButton(
+              viewModel: RoundedButtonViewModel(title: _viewModel.actionText),
+              style: RoundedButtonStyle.largeRoundedButtonStyle(),
+            ),
+          );
+  }
+
+  Padding _buildDetailText() {
+    return Padding(
+            padding: _style.detailTextEdgePadding,
+            child: Row(
```

Comment:
Same here, you can achieve this with a `Container` with a `width: double.infinity`